src/Microsoft.Data.Analysis/DataFrame.IO.cs

```diff
@@ -174,31 +172,12 @@ private static DataFrameColumn CreateColumn(Type kind, string[] columnNames, int
             return ret;
         }
 
-        /// <summary>
-        /// Reads a seekable stream of CSV data into a DataFrame.
-        /// Follows pandas API.
-        /// </summary>
-        /// <param name="csvStream">stream of CSV data to be read in</param>
-        /// <param name="separator">column separator</param>
-        /// <param name="header">has a header or not</param>
-        /// <param name="columnNames">column names (can be empty)</param>
-        /// <param name="dataTypes">column types (can be empty)</param>
-        /// <param name="numberOfRowsToRead">number of rows to read not including the header(if present)</param>
-        /// <param name="guessRows">number of rows used to guess types</param>
-        /// <param name="addIndexColumn">add one column with the row index</param>
-        /// <param name="encoding">The character encoding. Defaults to UTF8 if not specified</param>
-        /// <returns><see cref="DataFrame"/></returns>
-        public static DataFrame LoadCsv(Stream csvStream,
+        private static DataFrame ReadCsvLinesIntoDataFrame(IEnumerable<string> lines,
```

Comment:
This method is just a refactor of the logic in `LoadCsv` with 1 simple change: The first parameter here is an `IEnumerable<string>`.